src/components/LuxMenuBar.vue

```diff
@@ -591,7 +594,6 @@ export default {
 .dark {
   a,
   .lux-submenu-toggle {
-    &:hover,
```

Comment:
On hover in dark theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-black);
}